pkg/controller/operator.go

```diff
@@ -123,14 +127,21 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 		"clusterconfig": clusterConfigGatherer,
 	})
 	statusReporter.AddSources(periodic.Sources()...)
-	go periodic.Run(4, ctx.Done())
+
+	// check we can read IO container status and we are not in crash loop
+	err = wait.PollImmediate(20*time.Second, wait.Jitter(s.Controller.Interval/12, 0.1), isRunning(ctx, gatherKubeConfig))
```

Comment:
This will delay first gathering when running locally, because we usually don't set `POD_NAMESPACE` and `POD_NAME` env vars. But it shouldn't be long given the fact that the `s.Controller.Interval` is locally 5min.